### PR TITLE
IdleNotifier: change isAttackable to support deadman mode

### PR DESCRIPTION
**Pull request by michaelcreatesstuff**
In Deadman mode, anyone from lvl 3-126 is attackable.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java [hunk 1]
```diff
@@ -55,10 +55,17 @@ public static int getWildernessLevelFrom(WorldPoint point)
 	public static boolean isAttackable(Client client, Player player)
 	{
 		int wildernessLevel = 0;
-		if (!(client.getVar(Varbits.IN_WILDERNESS) == 1 || WorldType.isPvpWorld(client.getWorldType())))
+		if (!(client.getVar(Varbits.IN_WILDERNESS) == 1
+				|| WorldType.isPvpWorld(client.getWorldType())
+				|| client.getWorldType().contains(WorldType.DEADMAN)
```
Comment (xKylee) [inline runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java]: This should be `WorldType.isDeadmanWorld(client.getWorldType())` to be inline with the other WorldType calls.

File: runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java [hunk 2]
```diff
@@ -55,10 +55,17 @@ public static int getWildernessLevelFrom(WorldPoint point)
...
+		))
 		{
 			return false;
 		}
+		if (client.getWorldType().contains(WorldType.DEADMAN))
```
Comment (xKylee) [inline runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java]: This should be `WorldType.isDeadmanWorld(client.getWorldType())` to be inline with the other WorldType calls.

File: runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java (hunk 1 above)
Comment (michaelcreatesstuff) [reply runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java]: Done

File: runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java (hunk 2 above)
Comment (michaelcreatesstuff) [reply runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java]: Done

**Pull request merged**